nautobot/docs/user-guide/feature-guides/getting-started/creating-devices.md

```diff
@@ -73,25 +73,46 @@ To create a new Device Type:
 
 ![Create a device type](../images/getting-started-nautobot-ui/4-create-device-type.png)
 
-## Creating a Site
+## Creating a Location Type
 
-How you choose to employ sites when modeling your network may vary depending on the nature of your organization, but generally a site will equate to a building or campus.
-For example, a chain of banks might create a site to represent each of its branches, a site for its corporate headquarters, and two additional sites for its presence in two co-location facilities.
+To create a location, you need to specify a Location Type for the location.
```

Comment:
Something like:

```suggestion
To create a location, you need to specify a Location Type for the location. The Location Type(s) that you define for your network may vary depending on the nature of your organization and its needs, but might include for example "region", "city", "site", "building" and/or "room". Take some time to think about this before you begin entering data as it will be a fundamental aspect of how you record your network in Nautobot.
```